src/utils/loans.ts

```diff
@@ -0,0 +1,159 @@
+import { Bitcoin, MonetaryAmount } from "@interlay/monetary-js";
+import Big from "big.js";
+import { newMonetaryAmount } from "..";
+import { LoanAction, CurrencyExt, LoanAsset, LoanPosition, TickerToData } from "../types";
+
+const calculateTotalBorrowedBtcChange = (
+    action: LoanAction,
+    currentTotalBorrowedBtc: MonetaryAmount<Bitcoin>,
+    actionAmountBtc: MonetaryAmount<Bitcoin>
+): MonetaryAmount<Bitcoin> => {
+    switch (action) {
+        case "borrow":
+            return currentTotalBorrowedBtc.add(actionAmountBtc);
+        case "repay":
+            return currentTotalBorrowedBtc.sub(actionAmountBtc);
+        default:
+            return currentTotalBorrowedBtc;
+    }
+};
+
+const calculateCollateralAmountBtc = (
```

Comment:
Naming between this method and `calculateTotalBorrowedBtcChange` (ends with "Change") above seems to be inconsistent. I might be missing something, though.

Reply:
You are right, changed naming to make it consistent :)